BDArmory/Extensions/VesselExtensions.cs

```diff
@@ -89,7 +89,7 @@ public static float GetRadius(this Vessel vessel, Vector3 fireTransform = defaul
 #if DEBUG
                 if (radius < bounds.x / 2f && radius < bounds.y / 2f && radius < bounds.z / 2f) Debug.LogWarning($"DEBUG Radius {radius} of {vessel.vesselName} is less than half its minimum bounds {bounds}");
 #endif
-                return Mathf.Max(radius, Mathf.Max(Mathf.Max(vessel.vesselSize.x, vessel.vesselSize.y), vessel.vesselSize.z) / 2f); // clamp bounds to vesselsize
+                return Mathf.Min(radius, (Mathf.Max(Mathf.Max(vessel.vesselSize.x, vessel.vesselSize.y), vessel.vesselSize.z) / 2f) * 1.41f); // clamp bounds to vesselsize in case of Bounds erroneously reporting vessel sizes that are impossibly large
```

Comment:
Er, my bad, that should be `sqrt(3)` for the diagonal across a cube, not `sqrt(2)`.